api-gateway/src/main/resources/static/scripts/auth/update-role-form/role-update.service.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+angular.module('userModule').service('UserService', [function() {
+    this.getAvailableRoles = function() {
+        return ['ADMIN', 'VET', 'OWNER'];
```

Comment:
Does this normally work with the default preexisting roles? New ones would prove to be an issue no?